laborator/content/reprezentare-numere/README.md

```diff
@@ -72,7 +72,7 @@ Operația inversă, conversia din binar în hexazecimal se poate face convertind
 
 Sistemul hexazecimal este utilizat pentru a reprezenta adresele la care se află datele în memorie și pentru a vizualiza aceste date într-un mod mai ușor de interpretat decât o secvență compusă doar din 0 și 1. Imaginea de mai jos oferă un exemplu în acest sens:
 
-![Hartă memorie](https://www.digital-detective.net/wp-content/uploads/2018/08/Hex-View.png)
```

Comment:
Imaginile de pe alte site-uri ar trebui să fie creditate corespunzător.

Reply:
Ati putea elabora cum s-ar dori facut acest lucru? E nevoie de o bibliografie la sfarsitul fisierului?

Reply:
Poți adăuga în paranteze un link la site și textul "preluat de pe Digital Detective", sau ceva similar. Nu m-am gândit exact cum, dar e bine să existe ceva.

Reply:
E in regula asa?

Reply:
Da, este OK. Dar încearcă să nu faci foarte des force push, e scary să vezi mesajul ăla în general, pentru că în general înseamnă că se modifică istoricul. Adaugă commit-uri mici cu fiecare modificare, nu trebuie commit-uri mari (în general chiar sunt descurajate).